### ISSUE TITLE
Confusion about Experimental Discrepancies

### ISSUE THREAD

**Issue by DevKPro** [YiF-Zhang/RegionProxy#15]
Hello, author. I ran your open-source code on the Cityscapes dataset in my single GPU (3080Ti) environment, and the program ran smoothly. Since the paper used 8 GPUs, and I used only one, I accordingly reduced the learning rate to 1/8 of the original value. However, the experimental results are far from the results provided in your paper. Could you please explain the reason for this? I don't believe that single GPU execution is the main cause of this issue. Are there any other possibilities?
The test results after running 80k iterations of the code are as follows:
`
[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 500/500, 4.0 task/s, elapsed: 126s, ETA:     0s

2024-02-04 02:55:51,145 - mmseg - INFO - per class results:
2024-02-04 02:55:51,147 - mmseg - INFO - 
+---------------+-------+-------+
|     Class     |  IoU  |  Acc  |
+---------------+-------+-------+
|      road     | 90.24 | 96.64 |
|    sidewalk   | 39.15 | 51.54 |
|    building   | 75.24 | 91.19 |
|      wall     | 15.67 | 17.24 |
|     fence     |  2.16 |  2.39 |
|      pole     |  6.68 |  7.13 |
| traffic light |  0.06 |  0.06 |
|  traffic sign | 11.86 | 12.65 |
|   vegetation  | 81.37 | 90.16 |
|    terrain    | 44.97 |  57.3 |
|      sky      | 86.01 | 96.76 |
|     person    | 35.78 | 55.09 |
|     rider     |  0.0  |  0.0  |
|      car      | 67.18 | 86.92 |
|     truck     |  0.0  |  0.0  |
|      bus      |  0.35 |  0.35 |
|     train     |  0.0  |  0.0  |
|   motorcycle  |  0.0  |  0.0  |
|    bicycle    | 19.81 | 23.84 |
+---------------+-------+-------+
2024-02-04 02:55:51,147 - mmseg - INFO - Summary:
2024-02-04 02:55:51,148 - mmseg - INFO - 
+-------+-------+-------+
|  aAcc |  mIoU |  mAcc |
+-------+-------+-------+
| 85.38 | 30.34 | 36.28 |
+-------+-------+-------+
2024-02-04 02:55:51,149 - mmseg - INFO - Exp name: regproxy-s16-sub4+implicit-mid-2+768x768+80k+adamw-poly+cityscapes.py
2024-02-04 02:55:51,150 - mmseg - INFO - Iter(val) [500]        aAcc: 0.8538, mIoU: 0.3034, mAcc: 0.3628, IoU.road: 0.9024, IoU.sidewalk: 0.3915, IoU.building: 0.7524, IoU.wall: 0.1567, IoU.fence: 0.0216, IoU.pole: 0.0668, IoU.traffic light: 0.0006, IoU.traffic sign: 0.1186, IoU.vegetation: 0.8137, IoU.terrain: 0.4497, IoU.sky: 0.8601, IoU.person: 0.3578, IoU.rider: 0.0000, IoU.car: 0.6718, IoU.truck: 0.0000, IoU.bus: 0.0035, IoU.train: 0.0000, IoU.motorcycle: 0.0000, IoU.bicycle: 0.1981, Acc.road: 0.9664, Acc.sidewalk: 0.5154, Acc.building: 0.9119, Acc.wall: 0.1724, Acc.fence: 0.0239, Acc.pole: 0.0713, Acc.traffic light: 0.0006, Acc.traffic sign: 0.1265, Acc.vegetation: 0.9016, Acc.terrain: 0.5730, Acc.sky: 0.9676, Acc.person: 0.5509, Acc.rider: 0.0000, Acc.car: 0.8692, Acc.truck: 0.0000, Acc.bus: 0.0035, Acc.train: 0.0000, Acc.motorcycle: 0.0000, Acc.bicycle: 0.2384
`
